src/screens/ProfileScreen/index.js

```diff
@@ -70,8 +74,6 @@ import {setMyProfileFeed} from '../../context/actions/myProfileFeed';
 import {useAfterInteractions} from '../../hooks/useAfterInteractions';
 import {useUpdateClientGetstreamHook} from '../../utils/getstream/ClientGetStram';
 import {withInteractionsManaged} from '../../components/WithInteractionManaged';
-import ShadowFloatingButtons from '../../components/Button/ShadowFloatingButtons';
-import ImageCompressionUtils from '../../utils/image/compress';
 
 const {width} = Dimensions.get('screen');
 
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `ShadowFloatingButtons` and `ImageCompressionUtils` have been removed. If they are still being used in the code, this will cause a reference error.
```diff
- import ShadowFloatingButtons from '../../components/Button/ShadowFloatingButtons';
- import ImageCompressionUtils from '../../utils/image/compress';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->